.babelrc

```diff
@@ -51,7 +51,8 @@
       {
         "root": "./src",
         "alias": {
-          "vet360": "./src/platform/user/profile/vet360"
+          "vet360": "./src/platform/user/profile/vet360",
+          "#profile": "./src/applications/personalization/profile"
```

Comment:
@department-of-veterans-affairs/frontend-review-group

Reply:
I realize that this `#` might look a little weird, but it is there for a reason - explained in this [blog post](https://medium.com/bootstart/why-you-should-use-babel-resolvers-210615fc41d)

Its purpose is to indicate that it is a 'special' import and that it is unrelated to node_modules. As Erik mentioned above, we have plans to do the same for the `vet360` alias.

Reply:
Also curious if people have ideas for a different alias prefix. I swear we talked about this maybe a year ago and there were some ideas thrown out. 🤔

Reply:
I'm not a super fan of the `#` myself since that doesn't seem especially intuitive to me.

I looked around a bit to see how other folks manage this.`nx` uses or encourages (or something) the use of `import something from '@myorg/place';`. I couldn't find the documentation for that, but you can see the examples [in the tutorial](https://nx.dev/latest/react/tutorial/08-create-libs).

How do you feel about `@va/profile` instead of `#profile` to make it clearer that it's not a node module? (That's the purpose of this, right?)

Reply:
@cvalarida  @erikphansen  Do you guys think it might be confusing if we don't add 'app' in the alias? I am afraid that when we also do `@va/vet360` (which lives in platform), developers might be confused and think the `@va` points to the platform folder? I could do `@va/apps/profile` or `@va/profile-app` as well

If it doesn't sound confusing, I am totally good with changing it to `@va/profile` 👍

Reply:
But `@va/stuff` _could_ be a valid node module. True that `#` isn't intuitive. I'd argue that no character is an intuitive indicator that it's a Babel alias 🤔  But it serves as an indicator that there is something different about this import. And it'll become intuitive after we get used to whatever we pick 😆  Anyway yes the _purpose_ is to make it clear that this isn't a node module.

Reply:
`@namespace` is fine with me. @department-of-veterans-affairs is the npm namespace and I don't see that changing to @va. `/namespace` works for me too as long as the folder doesn't have an overly generic name.

Reply:
If the purpose if to make it clear we aren't importing from `node_modules` using `@` at the start of our alias is still potentially confusing to someone who's going to wonder "why am I not seeing any `@va` modules in `node_modules`?" And using `/namespace` would be even more confusing since that will _hopefully_ be a stand in for the `src/namespace/` dir.

Reply:
Since a couple people want to use `@` as the prefix, how about using `@@` instead? So `"@@profile": "./src/applications/personalization/profile"` and then `import Module from '@@profile/Module'`?

Reply:
I think that `@@` is a good solution, it will definitely stand out. I could only find [one other instance](https://dmitripavlutin.com/detailed-overview-of-well-known-symbols/) of where the `@@` is used in JS, but we're not using it this way in our codebase, so it should not be confusing

Reply:
@rianfowler I updated it to `@@namespace` - hope this makes sense to you as well 😄